### DISCUSSION TITLE
displaying-complex-content-inside-jekyll-tables/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#40] [Comments]
# displaying-complex-content-inside-jekyll-tables/

As anyone who writes in Markdown knows, it’s a very concise and easy way to write almost all content. However, tables can be a little bit awkward, especially if it includes multi-line content like a list! Here’s a quick comparison of techniques that can be used to solve this, and whether they work in plain Markdown, GitHub, and Jekyll blogs.

https://blog.jakelee.co.uk/displaying-complex-content-inside-jekyll-tables/

**Comment by dustin-ctg** [comment]
This is exactly what I was looking for, to help me write a specification doc for a project I'm working on. Thank you so much!
